docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2298,6 +2298,8 @@ preserved, it will be explicitly stated in this document. The
 C++ API supports comparing messages while treating repeated fields as sets, so
 that different orderings of the same elements are considered equal. This method
 of comparing Protobuf messages may come at a cost in performance.
+The `backup_replicas` must be returned in order when read to preserve the
+hierarchy of highest-preferred backup replicas.
```

Comment:
@jafingerhut recommended using another term instead of hierarchy, e.g. sequence or order

Reply:
+1